pkg/workload/workload.go

```diff
@@ -51,6 +51,12 @@ func NewInfo(w *kueue.Workload) *Info {
 	}
 }
 
+func (i *Info) Update(wl *kueue.Workload, cqName string) {
```

Comment:
https://github.com/kubernetes-sigs/kueue/pull/256 
if we forbid to update the podset in the future, we may not need to calculate the resource here?

Reply:
You are right. Once #256 merges, maybe we should make sure in the workload controller if the UID changed and issue a Delete and Create instead of a single Update.

Reply:
OK. we only update `Obj` in `Update` method. 


> maybe we should make sure in the workload controller if the UID changed and issue a Delete and Create instead of a single Update.

Do I need to create an issue to track this?

Reply:
> Do I need to create an issue to track this?

@cmssczy  Yes. Please to create an issue for this. Thanks.

Reply:
I have a question about

> make sure in the workload controller if the UID changed

In what scenarios will this happen? UID may be a read-only parameter.

Reply:
I'm not sure if this could still happen. But in the past we have seen the k8s client merge a Delete event and a Create event, for an object with the same namespace/name, into a single Update event.
In that scenario, you would see that the UID is theoretically updated. But in reality it's just a new object.

Reply:
Create an issue to track this: https://github.com/kubernetes-sigs/kueue/issues/261